libs/isograph-react/src/EntrypointReader.tsx

```diff
@@ -1,23 +1,27 @@
-import * as React from 'react';
+import * as React from "react";
```

Comment:
this was an unintended change but it seems that it's how your Prettier is configured here